src/dos/dos_memory.cpp

```diff
@@ -18,23 +18,90 @@
 
 #include "dosbox.h"
 
-#include "mem.h"
 #include "dos_inc.h"
+#include "enum.h"
+#include "mem.h"
 
-#define UMB_START_SEG 0x9fff
+enum class McbFaultStrategy { deny, repair, report, allow };
 
-static uint16_t memAllocStrategy = 0x00;
+// Constants
+// ~~~~~~~~~
 
-static void DOS_CompressMemory(void) {
-	uint16_t mcb_segment=dos.firstMCB;
+// MCB - DOS Memory Control Block Format
+// Offset Size   Description
+// 00     byte   'M' 4Dh  member of a MCB chain, (not last)
+//               'Z' 5Ah  indicates last entry in MCB chain
+// - the 'M' and 'Z' are said to represent Mark Zbikowski
+constexpr uint8_t mcb_type_M = {0x4d};
+constexpr uint8_t mcb_type_Z = {0x5a};
+
+// Upper member block starting segment
+constexpr uint16_t UMB_START_SEG = {0x9fff};
+
+static uint16_t memAllocStrategy = {0x00};
+
+constexpr uint16_t max_allowed_faults = {100};
```

Comment:
Why the curly braces around all these simple value assignments? It's just noise, no?

Reply:
Indeed; knocked out.